Gemfile

```diff
@@ -36,6 +36,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'listen', '~> 3.0.5'
```

Comment:
what's the purpose of adding this gem?

Reply:
It is required by our existing configuration: https://github.com/sul-dlss/dor_indexing_app/blob/b12e7a5ce02d9750ef27abb609bf2d66ece89e33/config/environments/development.rb#L40  I had the gem locally, so I didn't know it was a problem until building in a clean (docker) environement.

Reply:
ah, got it, figured something like code reloading.  thanks for the explanation!